app/views/static_pages/about_us.html.erb

```diff
@@ -34,10 +34,11 @@
 
       <h2 class="fontsize-h3"><%= _('Getting started') %></h2>
 
-        <p><%= _("If you have an account please sign in and start creating or editing your DMP.") %></p>
+        <p><%= _("If you have an account, please sign in and start creating or editing your DMP.") %></p>
+        <!--
         <p><%= _("If you do not have a %{application_name} account, click on")  % {application_name: ApplicationService.application_name} %> <a href="<%= root_path %>"><%= _('Sign up') %></a> <%= _("on the homepage.") %></p>
```

Comment:
We were using the ``format`` method in other instances, we should decide on an approach and stick with it. I am fine with both. Probably this simplified way is better though.

This is something else we can bring up in the dev meeting once we catch up to version 4.

Reply:
I agree, would be nice to have it all one way. I think we originally tried this approach for the landing page, but Rubocop wanted the format method.

Reply:
Also, I just noticed that one of these help links is broken on production https://github.com/portagenetwork/roadmap/issues/412.
For now, I just made a small edit to the existing code https://github.com/portagenetwork/roadmap/pull/409/commits/57f5d256b7ef409211c97bf72f86af3da122254c. But the reformatting you pointed out should be addressed soon as well.

Reply:
We are using the simplified version on all our ERB files. Let's keep it as it is for now unless upstrem changes that.